Ecosystem Partners/getting-started-with-jboss-blueprint.md

```diff
@@ -0,0 +1,153 @@
+{{{
+  "title": "Getting Started with JBoss - Blueprint",
+  "date": "05-05-2015",
+  "author": "<a href='https://twitter.com/KeithResar'>@KeithResar</a>",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+
+
+### Description
+
+<img src="//d33np9n32j53g7.cloudfront.net/assets/stacks/jboss/img/jboss-stack-110x117-12361d85ff78442ab801caef8baf4d63.png" style="border:0;float:right;max-width:250px">
```

Comment:
the alt attribute should contain a brief description of that the Image is displaying. For example the image tag displaying the CLC logo would have and alt attribute of 'CLC logo' where as a screenshot displaying something would have "Screenshot displaying feature XYZ"